docs/device_automation_index.md

```diff
@@ -8,3 +8,42 @@ Device Automations provide users with a device-centric layer on top of the core
 Integrations can hook into this system by exposing functions to generate the pre-defined triggers, conditions, actions and having functions that can listen for the triggers, check the condition and execute the action.
 
 Device automations are not exposing extra functionality but are a way for users to not have to learn new concepts. Device automations are using events, state and service helpers under the hood.
+
+### Secondary device automations
+
+Some devices may expose a lot of device automation. To not overwhelm the user, a device automation can be marked as secondary. A trigger which is marked as secondary will still be shown to the user, but may be shown after other device automations or may require the user to select a "show more" option or similar.
```

Comment:
Is this only true for triggers and not for actions and conditions?

Reply:
Oops, it's indeed true for actions, conditions and triggers!